lib/builtin.pm

```diff
@@ -154,6 +155,54 @@ Returns the basic container type of the referent of a reference, or C<undef>
 for a non-reference. This is returned as a string in all-capitals, such as
 C<ARRAY> for array references, or C<HASH> for hash references.
 
+=head2 created_as_string
+
+    $bool = created_as_string($val);
+
+Returns a boolean representing if the argument value was originally created as
+a string. It will return true for any scalar expression whose most recent
+assignment or modification was of a stringy nature - such as assignment from a
```

Comment:
Since this documentation will presumably be around for a long time to come, I'd suggest `s/stringy/string-like/g` to give a less casual tone.

Reply:
Done in 8c82e520c2